rpe/resources/gcp.py

```diff
@@ -173,67 +174,71 @@ def type(self):
     # simplify the structure of our policies
     def full_resource_name(self):
 
-        method = getattr(self.service, self.get_method)
-        uri = method(**self._get_request_args()).uri
+        if self._full_resource_name is None:
+
```

Comment:
would it make sense to pull this suite into a function?  It's kind of hard to read that much under an `if`

Reply:
Hows this version? I wish we could make the full_resource_name a property to fit in with the project_id, organization, labels etc. That would be a breaking change though